Makefile

```diff
@@ -118,6 +120,32 @@ ifeq "$(LXD_OFFLINE)" ""
 endif
 	swagger generate spec -o doc/rest-api.yaml -w ./lxd -m
 
+.PHONY: update-embedded-full-documentation
+update-embedded-full-documentation: doc
+	rsync -a --delete doc/html/ lxd/gendocs/html/
+	# Remove the .doctrees directory, as it is useless for us and will reduce the size.
+	rm -Rf lxd/gendocs/html/.doctrees
+
+	# Minify content
+	@echo "Minifying HTML files..."
+	@find lxd/gendocs/html/ -name '*.html' -type f -print0 | xargs -0 -n 1 -P `nproc` -I {} $(HTMLMINIFY) -o {} {}
+
+	@echo "Minifying CSS files..."
+	@find lxd/gendocs/html/ -name '*.css' -type f -print0 | xargs -0 -n 1 -P `nproc` -I {} postcss --use cssnano -o {} {}
+
+	@echo "Minifying JS files..."
+	@find lxd/gendocs/html/ -name '*.js' -type f -print0 | xargs -0 -n 1 -P `nproc` -I {} $(JSMINIFY) -o {} {}
+
+	@echo "Minifying SVG files..."
+	@find lxd/gendocs/html/ -name '*.svg' -type f -print0 | xargs -0 -n 1 -P `nproc` -I {} svgo {} --output={}
+
+	@echo "Optimizing PNG files..."
+	@find lxd/gendocs/html/ -name '*.png' -type f -print0 | xargs -0 -n 1 -P `nproc` -I {} optipng -o7 {}
```

Comment:
AFAIK, PNG should not be `gzip` as that compression/optimization is done by `optipng` already.